examples/spring-data-graphql/pom.xml

```diff
@@ -85,14 +85,21 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>com.graphql-java</groupId>
+            <groupId>com.graphql-java-kickstart</groupId>
```

Comment:
Looks like this change doesn't work for Spring Data 1.11.x:

```
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean found for dependency [graphql.kickstart.servlet.context.GraphQLServletContextBuilder]: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=false)}
```

Reply:
Yeah, the newer versions included in the upgraded `graphql-java-kickstart` no longer works with boot 1.11.*. I suggest dropping the profile (or faking it into 2.0 as is already done in https://github.com/Blazebit/blaze-persistence/blob/020ebd73b188613d824bdda8f18172365013a062/examples/spring-data-dgs/pom.xml#L206).

Reply:
Faking it sounds fine to me, can you please make the changes and then squash everything? :)

Reply:
I realized that dropping the profile would still lead to running the test, although with the default profile version (2.7 in this case). So it really doesn't matter and I faked it into 2.0.